src/OrchardCore.Modules/OrchardCore.Resources/Startup.cs

```diff
@@ -39,3 +40,13 @@ public override void ConfigureServices(IServiceCollection serviceCollection)
         serviceCollection.AddScoped<IResourcesTagHelperProcessor, ResourcesTagHelperProcessor>();
     }
 }
+
+[RequireFeatures("OrchardCore.Liquid")]
+public sealed class ResourcesLiquidStartup : StartupBase
+{
+    public override void ConfigureServices(IServiceCollection services)
```

Comment:
I don’t think this is the right approach. If you look at line 24, you’ll see that we’re always registering Liquid-related services without needing to explicitly check for `OrchardCore.Liquid`. This is because Liquid support is automatically included when `.AddOrchardCms()` is called in the web project. In other words, the Liquid services are available even if `OrchardCore.Liquid` is not explicitly enabled.

A better solution would be to introduce a new feature, `OrchardCore.Liquid.Core`, which would register the necessary Liquid services. This feature would be enabled through dependencies and could be used in the current context. We can then move all Liquid-related configuration into a new startup class dedicated to this feature. This approach would allow us to cleanly encapsulate the registration of Liquid services, as shown here: [Liquid Service Registration](https://github.com/OrchardCMS/OrchardCore/blob/f2623140e1f2f07b7852068345ffe8380bc1a968/src/OrchardCore/OrchardCore.DisplayManagement.Liquid/OrchardCoreBuilderExtensions.cs#L25-L52).

Furthermore, the current line [here](https://github.com/OrchardCMS/OrchardCore/blob/f2623140e1f2f07b7852068345ffe8380bc1a968/src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/ServiceExtensions.cs#L31) would be replaced with `.AddGlobalFeatures("OrchardCore.Liquid.Core")`.

This would simplify the code and improve modularity, ensuring that Liquid services are only registered when the new feature is explicitly enabled.